public/assets/css/module.css

```diff
@@ -1,16 +1,15 @@
-
 .code-area {
   background-color: #333333;
   color: #fff;
+  height: 77% !important;
```

Comment:
Is this `!important` really needed?, I mean if we can get rid of it. If this class is overriding styles from a bootstrap class for example, maybe we should check if we really need that bootstrap class, if so, what do you think about making a selector with higher specificity to override the bootstrap styles?.